projects/core/src/model/cart.model.ts

```diff
@@ -9,12 +9,18 @@ import {
 import { CostCenter } from './org-unit.model';
 import { Price, Promotion } from './product.model';
 
+/**
+ * @deprecated since 5.0, use cart lib instead
+ */
 export interface PromotionResult {
   consumedEntries?: PromotionOrderEntryConsumed[];
   description?: string;
   promotion?: Promotion;
 }
 
+/**
+ * @deprecated since 5.0, use cart lib instead
+ */
 export enum PromotionLocation {
```

Comment:
Let's create a new ticket to investigate whether this one can be removed, because I remembered we put promotion data directly into cart/order entry.

Reply:
Let's create a ticket to investigate it.

Reply:
Ok